docs/experiments/request_locking.mdx

```diff
@@ -6,6 +6,21 @@ description: Parallelize crawlers with ease using request locking
 
 import ApiLink from '@site/src/components/ApiLink';
 
+:::tip Release announcement
+
+As of **May 2024** (`crawlee` version `3.9.0`), this experiment is now enabled by default! With that said, if you encounter issues you can:
```

Comment:
let's do a feature release since we are adjusting the docs

```suggestion
As of **May 2024** (`crawlee` version `3.10.0`), this experiment is now enabled by default! With that said, if you encounter issues you can:
```